src/app/pages/faq/faq.component.html

```diff
@@ -83,7 +89,7 @@ <h2 class="mb-0">
       <!-- Answer -->
       <div id="s1q4c" class="collapse" aria-labelledby="s1q4h" data-parent="#faq-s1">
         <div class='card-body card-align'>
-          Go to <a routerLink="/your-virtrolio">your Virtrolio</a> and click on 'Sharing Link,', then generate a new
+          Go to <a routerLink="/your-virtrolio">your Virtrolio</a> and click on 'Sharing Link,' then generate a new
```

Comment:
You removed the wrong comma.
```suggestion
          Go to <a routerLink="/your-virtrolio">your Virtrolio</a> and click on 'Sharing Link', then generate a new
```